microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -101,10 +101,19 @@ Issued: 2023-02-16
 For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel8/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel8]
 
 [id="microshift-4-12-4-dp"]
-=== RHSA-2023:0769 - {product-title} 4.12.4 bug fix
+=== RHSA-2023:0769 - {product-title} 4.12.4 bug fix and security update
 
 Issued: 2023-02-20
 
-{product-title} release 4.12.4 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0769[RHSA-2023:0769] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:0768[RHBA-2023:0768] advisory.
+{product-title} release 4.12.4, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0769[RHSA-2023:0769] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:0768[RHBA-2023:0768] advisory.
 
 For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel8/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel8]
+
+[id="microshift-4-12-5-dp"]
+=== RHBA-2023:0893 - {product-title} 4.12.5 bug fix update
+
+Issued: 2023-02-28
+
+{product-title} release 4.12.5 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:0893[RHBA-2023:0893] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:0890[RHSA-2023:0890] advisory.
```

Comment:
Both of the links for errata lead to a 404 for me. Are the errata published?

Reply:
Not yet, they will be live tomorrow when the release goes out. Thanks! I forgot to mention this!